src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.PolicyInsights" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.GuestConfiguration" version="0.9.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="2.0.0-preview" targetFramework="net452" />
```

Comment:
You can't take the ResourceManager or PolicyInsights dependencies in a separate module

Reply:
as per our discussion, retaining Microsoft.Azure.Management.ResourceManager.dll for now